jx/scripts/ci-github.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+export GH_CREDS_PSW="$(jx step credential -s WHAT_TO_PUT_HERE)"
+export JENKINS_CREDS_PSW="$(jx step credential -s  test-jenkins-user)"
+export GKE_SA="$(jx step credential -s gke-sa)"
+
+# fix broken `BUILD_NUMBER` env var
+export BUILD_NUMBER="$BUILD_ID"
+
+JX_HOME="/tmp/jxhome"
+KUBECONFIG="/tmp/jxhome/config"
+
+mkdir -p $JX_HOME
+
+jx --version
+jx step git credentials
+
+gcloud auth activate-service-account --key-file $GKE_SA
+
+# lets setup git 
+git config --global --add user.name JenkinsXBot
+git config --global --add user.email jenkins-x@googlegroups.com
+
+echo "running the BDD tests with JX_HOME = $JX_HOME"
+
+jx step bdd --config jx/bdd/staticjenkins.yaml --gopath /tmp --git-provider=github --git-username WHAT_TO_PUT_HERE --git-api-token $GH_CREDS_PSW --default-admin-password $JENKINS_CREDS_PSW --no-delete-app --no-delete-repo --tests install --tests test-create-spring
```

Comment:
@jstrachan - do you know what `--git-username`  you want to use for tests against github (as opposed to GHE) for the terraform tests?